Module Training Updates (#175)
github-campus-advisors/Campus-Advisor-Training: Module 2/Module 2.3 Assignment.md

# Module 2.3 Assignment

> In this lesson, you learned how branches are used in the GitHub workflow. To complete this assignment, you’ll change this file on a different branch using the command line from your local repository by following the instructions below:

Reviewer: ```suggestion
In this lesson, you learned more about how branches are used in the GitHub workflow. To complete this assignment, you’ll use the command line to create a new branch and change this file in your local repository by following the instructions below:
```

> 1. Create a new branch named `assignment-edit` in your repository using the command line

Reviewer: ```suggestion
1. Create a new branch named `assignment-edit` in your local repository using the command line
```
